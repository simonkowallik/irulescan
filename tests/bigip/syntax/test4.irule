when RULE_INIT priority 500 {
  for   
 {set x 0}   
    {$x<10}   
{incr x}   {
    log local0. "for x is [expr $x]"
}
}