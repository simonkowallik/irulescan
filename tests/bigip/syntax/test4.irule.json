{
  "warning": [],
  "dangerous": [
    "Dangerous unquoted expr at `$x` in `expr $x`"
  ]
}
