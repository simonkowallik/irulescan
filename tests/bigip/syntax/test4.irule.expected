DANGEROUS: Dangerous unquoted expr at `$x` in `expr $x`

